src/main/java/seedu/address/model/internship/InternshipItem.java

```diff
@@ -34,7 +40,13 @@
     private Set<Requirement> requirements = new HashSet<>();
 
     /**
-     * Every field must be present and not null.
+     * Constructs an InternshipItem object. Every field must be present and not null.
```

Comment:
I realized for constructors, in our main code, we use Creates, so maybe we could stick to the same javadocs for cohesiveness

Reply:
I think it really depends on what context is it being used in. There seems to be an almost equal amount of Creates and Constructs used in our code atm.